doc/source/cluster/cluster_under_construction/running-applications-on-ray-clusters/job-submission/jobs-package-ref.rst

```diff
@@ -0,0 +1,75 @@
+.. _ray-job-submission-api-ref-under-construction:
+
+Ray Job Submission API Reference
+================================
+
+For an overview with examples see :ref:`Ray Job Submission<jobs-overview>`.
+
+.. _ray-job-submission-cli-ref:
+
+Job Submission CLI
+------------------
+
+.. _ray-job-submit-doc:
+
+.. click:: ray.dashboard.modules.job.cli:submit
+   :prog: ray job submit
+
+.. warning::
+
+    When using the CLI, do not wrap the entrypoint command in quotes.  For example, use 
+    ``ray job submit --working_dir="." -- python script.py`` instead of ``ray job submit --working_dir="." -- "python script.py"``.
+    Otherwise you may encounter the error ``/bin/sh: 1: python script.py: not found``.
+
+.. _ray-job-status-doc:
+
+.. click:: ray.dashboard.modules.job.cli:status
+   :prog: ray job status
+   :show-nested:
+
+.. _ray-job-stop-doc:
+
+.. click:: ray.dashboard.modules.job.cli:stop
+   :prog: ray job stop
+   :show-nested:
+
+.. _ray-job-logs-doc:
+
+.. click:: ray.dashboard.modules.job.cli:logs
+   :prog: ray job logs
+   :show-nested:
+
+.. _ray-job-list-doc:
+
+.. click:: ray.dashboard.modules.job.cli:list
+   :prog: ray job list
+   :show-nested:
+
+.. _ray-job-submission-sdk-ref:
+
+Job Submission SDK
+------------------
+
+.. _job-submission-client-ref:
+
+JobSubmissionClient
+~~~~~~~~~~~~~~~~~~~
+
+.. autoclass:: ray.job_submission.JobSubmissionClient
+    :members:
+
+.. _job-status-ref:
+
+JobStatus
+~~~~~~~~~
+
+.. autoclass:: ray.job_submission.JobStatus
+    :members:
+
+.. _job-info-ref:
+
+JobInfo
+~~~~~~~
+
+.. autoclass:: ray.job_submission.JobInfo
+    :members:
```

Comment:
I think this will fail the linter because of duplicate definitions of the rst reference to `ray.job_submission.JobInfo`. You can solve this by adding a `:noindex:` tag as I did [here](https://github.com/ray-project/ray/blob/master/doc/source/cluster/cluster_under_construction/references/job-submission-apis.rst). If you add a TODO I'll make to remove these when we remove the old content.